lib/runtime-c-api/src/lib.rs

```diff
@@ -1241,7 +1241,7 @@ pub unsafe extern "C" fn wasmer_import_func_returns_arity(
 /// Frees memory for the given Func
 #[allow(clippy::cast_ptr_alignment)]
 #[no_mangle]
-pub extern "C" fn wasmer_import_func_destroy(func: *mut wasmer_import_func_t) {
```

Comment:
Should this one be `mut` since it is cast to `mut` and `drop` takes `&mut` ?

Reply:
Yeah, that's the only one I wasn't sure about. Thanks for the notice!